piet/src/samples/mod.rs

```diff
@@ -55,6 +56,7 @@ pub fn get<R: RenderContext>(number: usize) -> Result<SamplePicture<R>, BoxErr>
         13 => SamplePicture::new(picture_13::SIZE, picture_13::draw),
         14 => SamplePicture::new(picture_14::SIZE, picture_14::draw),
         15 => SamplePicture::new(picture_15::SIZE, picture_15::draw),
+        16 => SamplePicture::new(picture_16::SIZE, picture_16::draw),
```

Comment:
Also need to increase `pub const SAMPLE_COUNT: usize = 16;` to 17 -- it's a few lines before this.